service/src/main/resources/config/aou/test/SC2023Q3R1/entitygroup/t_device_person_occurrence.json

```diff
@@ -20,6 +20,11 @@
         "idPairsTable": { "table": "device_exposure" },
         "idPairsIdA": { "column": "device_concept_id" },
         "idPairsIdB":  { "column": "person_id" }
+      },
+      "groupToItems": {
```

Comment:
This property doesn't make sense here. The `type` above says `CRITERIA_OCCURRENCE`, not `GROUP_ITEMS`. I think you just want to leave this as is, but what were you trying to add here?

Reply:
Not sure... I do not understand this node... but hey it is working. Can you fix this? I do not understand the relationshipMapping. Need more documenttation

Reply:
Git it. removed `groupToItems` node